pkg/apis/provisioning/v1alpha2/provisioner_validation_test.go

```diff
@@ -89,6 +90,29 @@ var _ = Describe("Validation", func() {
 			}
 		})
 	})
+	Context("Taints", func() {
+		It("should succeed for valid taints", func() {
```

Comment:
Can you add a test to fail for invalid taint values?

Reply:
Good eye!